CMakeLists.txt

```diff
@@ -25,6 +29,7 @@ target_compile_definitions(furnace PRIVATE
         WIFI_PASSWORD=\"${WIFI_PASSWORD}\"
         CYW43_HOST_NAME=\"pico_furnace\"
         CONFIG_THERMO=${CONFIG_THERMO}
+        MAGNETRON_ON=${MAGNETRON_ON}
```

Comment:
Please, rename `MAGNETRON_ON` to `CONFIG_MAGNETRON`.
We prefer to use `CFLAGS` for project configuration options, because they get propagated to all executions of `gcc`. This is not a bug at this moment, but we already have two targets and with this change we would be passing the options to only this one, leaving behind the program at `native/`. Using `CFLAGS` for global configuration options just scales better and always does the right thing.

Changing configuration requires running `make clean`, but It's just what it is...